Client/lib/PONAPI/Client/UA/YAHC.pm

```diff
@@ -31,6 +31,11 @@ has scheme => (
     isa => 'Str',
 );
 
+has ssl_options => (
+    is  => 'rw',
```

Comment:
is there a reason why this attribute needs to be 'rw'? (I see the others there and think they should all be changed to 'ro')

Reply:
Yes, they could all be `ro`. I just followed the suit with `rw`.

Reply:
OK. I'll change them all in a separate commit.